### PR TITLE
Add osu! top player voting 2021 news post

### PR DESCRIPTION
**Pull request by hallowatcher**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

---

Video was added, as soon as it passes review it can be posted 👍

### CODE REVIEW

File: news/2021-12-27-clickers-choice-2021.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to join the hwchr projects discord server or contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
+
+[TOP 20 VOTING](https://choice.hwc.hr/vote/6?mtm_campaign=news-post-2021&mtm_kwd=top-20-voting) | [HOMEPAGE](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage) | [DISCORD](https://u.hwc.hr/discord))
```
Comment (ZeroPyrozen) [inline news/2021-12-27-clickers-choice-2021.md:25]:
```suggestion
[TOP 20 VOTING](https://choice.hwc.hr/vote/6?mtm_campaign=news-post-2021&mtm_kwd=top-20-voting) | [HOMEPAGE](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage) | [DISCORD](https://u.hwc.hr/discord)
```

Comment (ZeroPyrozen) [reply news/2021-12-27-clickers-choice-2021.md:25]: Change the web link to `https://choice.hwc.hr/vote/6` and `https://choice.hwc.hr` ?

Comment (hallowatcher) [reply news/2021-12-27-clickers-choice-2021.md:25]: Any chance I can leave them as-is? I would like to know how many people come from this news post by including a campaign name

Comment (hallowatcher) [reply news/2021-12-27-clickers-choice-2021.md]: Fixed the dangling parenthesis 👍

Comment (peppy) [reply news/2021-12-27-clickers-choice-2021.md]: Remove discord link and remove `mtm_kwd` whatever that is.

File: news/2021-12-27-clickers-choice-2021.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+layout: post
+title: "The Best osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+It's time for the community to decide the top 20 osu! players of 2021!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo-play and tournaments. Now that the year's coming to an end, we're bringing you the **9th iteration of the international top 20 voting**, courtesy of my latest project: [Clicker's Choice](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage)!
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:11]:
```suggestion
2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo play and tournaments. Now that the year's coming to an end, we're bringing you the **9th iteration of the international top 20 voting**, courtesy of my latest project: [Clicker's Choice](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage)!
```

File: news/2021-12-27-clickers-choice-2021.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+The Top 20 vote will be run on the Clicker's Choice website, which is already home to the top player voting for Germany, Poland, Canada, UK and many other countries. Clicker's Choice aims to make it easier for users to vote, while also making it simpler for the host to tabulate and evaluate results.
+
+As a voter, it's pretty straightforward - just log in through osu! and start voting! **Everything regarding the criteria and deadlines are on the voting page as well.** Here's a showcase of the entire process:
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:15]:
```suggestion
As a voter, it's pretty straightforward — just log in through osu! and start voting! **Everything regarding the criteria and deadlines are on the voting page as well.** Here's a showcase of the entire process:
```

File: news/2021-12-27-clickers-choice-2021.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to join the hwchr projects discord server or contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:23]:
```suggestion
If you have any further questions, feel free to join the hwchr projects Discord server or contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

File: news/2021-12-27-clickers-choice-2021.md (hunk 2 above)
Comment (Walavouchey) [reply news/2021-12-27-clickers-choice-2021.md]: i'm very sure it's not supposed to be hyphenated here

**Comment by peppy** [comment]
Post needs more thought.

**Comment by hallowatcher** [comment]
The mtm parameters stand for Matomo, a self-hosted tracking solution that was counting the clicks depending on the advertising campaign. I have the same for Reddit, for example. If it's not okay to track it this way, then I'll just have to track it by the Referer header

image

Otherwise, the changes look good. Let us know what needs more work @peppy 👍

**Comment by peppy** [comment]
WE don't do tracking of users. If you are to get featured on the news stream, we'd appreciate you also do not.

**Comment by Walavouchey** [comment]
[we have done this kind of formatting before](https://osu.ppy.sh/home/news/2021-11-10-osu!trivium-quiz-2021) and to me that looked pretty fine, although for this case it might be better to have only one link since both point to the same website (and thus not just put the link on its own)

image


image

**Comment by peppy** [comment]
The whole point of the post is to have people vote there right? If so, it should be the focus of the post, and not some link in a set of links.

Agree it should be the only link, there's no need to link to the home page.

**Comment by hallowatcher** [comment]
> WE don't do tracking of users. If you are to get featured on the news stream, we'd appreciate you also do not.

Just for clarification, it would just be counting the clicks, not the user. But I have no problem with omitting the campaign from the links

> we have done this kind of formatting before and to me that looked pretty fine, although for this case it might be better to have only one link since both point to the same website (and thus not just put the link on its own)

I used this post as a template, since mango included his Discord and forum post website, along the quiz link itself. But it's fine to just leave the 1 link then

File: news/2021-12-27-clickers-choice-2021.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
+
+This needs a better post format beucase it oesn't read well at all.. [TOP 20 VOTING](https://choice.hwc.hr/vote/6)
+
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:26]: maybe like this

```suggestion
But other than that, what are you waiting for? [Go ahead and vote!](https://choice.hwc.hr/vote/6)
```

Comment (hallowatcher) [reply news/2021-12-27-clickers-choice-2021.md]: Oh woops, I already changed it. what do you think of the version i committed?

**Comment by ZeroPyrozen** [comment]
You may add these as contents:
- Event timestamp (e.g. when the voting start and end? when the winner will be announced?)
- How user voting counted
- Rewards for the winners

File: news/2021-12-27-clickers-choice-2021.md (hunk 5 above)
Comment (Walavouchey) [reply news/2021-12-27-clickers-choice-2021.md]: This reads better considering paragraph flow, and the link doesn't need to include both sentences

Comment (hallowatcher) [reply news/2021-12-27-clickers-choice-2021.md]: done

File: news/2021-12-27-clickers-choice-2021.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
+
+But other than that, what are you waiting for? [Go ahead and vote!](https://choice.hwc.hr/vote/6)
+
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:26]:
```suggestion
```

**Comment by hallowatcher** [comment]
> You may add these as contents:
> 
>     * Event timestamp (e.g. when the voting start and end? when the winner will be announced?)
> 
>     * How user voting counted
> 
>     * Rewards for the winners

Thanks for the reply. Since this is a community voting, and things are bound to change, there's this line in the article:
`Everything regarding the criteria and deadlines are on the voting page as well.`

Then, in the voting page:
image

Maybe we can add the deadline to the news post, but everything else is of little immediate use to the reader IMO

**Comment by Walavouchey** [comment]
> Rewards for the winners

Does anything happen as a result of the votes? Is that just not decided yet? I believe this point would be very relevant for anyone reading.

**Comment by hallowatcher** [comment]
> > Rewards for the winners
> 
> Does anything happen as a result of the votes? Is that just not decided yet? I believe this point would be very relevant for anyone reading.

Good question, the only thing decided is that a video will follow just like every year, but I don't know anything about rewards. Either winging it and getting the top 5 supporter myself, or figure something out with Eph

**Comment by ZeroPyrozen** [comment]
> > > Rewards for the winners
> > 
> > 
> > Does anything happen as a result of the votes? Is that just not decided yet? I believe this point would be very relevant for anyone reading.
> 
> Good question, the only thing decided is that a video will follow just like every year, but I don't know anything about rewards. Either winging it and getting the top 5 supporter myself, or figure something out with Eph
> 
> But thinking about it, it would be cool to send out some personalized trophies to the winners - I'll look into it

No badges like Mapper Choice Award?

**Comment by hallowatcher** [comment]
> > > > Rewards for the winners
> > > 
> > > 
> > > Does anything happen as a result of the votes? Is that just not decided yet? I believe this point would be very relevant for anyone reading.
> > 
> > 
> > Good question, the only thing decided is that a video will follow just like every year, but I don't know anything about rewards. Either winging it and getting the top 5 supporter myself, or figure something out with Eph
> > But thinking about it, it would be cool to send out some personalized trophies to the winners - I'll look into it
> 
> No badges like Mapper Choice Award?

Ah, it never occurred to me to ask for badges, since this is the first time I'm hosting such event on my website (i.e. badge restrictions apply to first-time events)

**Comment by hallowatcher** [comment]
On second thought, there shouldn't be influential rewards that will affect the voting - e.g. a top player really wants a badge and starts spamming `Vote for me`, that isn't my vision at all

**Comment by megu-osu** [comment]
> > > > Rewards for the winners
> > > 
> > > 
> > > Does anything happen as a result of the votes? Is that just not decided yet? I believe this point would be very relevant for anyone reading.
> > 
> > 
> > Good question, the only thing decided is that a video will follow just like every year, but I don't know anything about rewards. Either winging it and getting the top 5 supporter myself, or figure something out with Eph
> > But thinking about it, it would be cool to send out some personalized trophies to the winners - I'll look into it
> 
> No badges like Mapper Choice Award?

there has never been a reward for a top player voting and there shouldnt be. mapper choice award makes sense as it is from a creation by that user which also benefitted the game. the purpose of the newspost is to promote the voting to a larger audience and should not be taken competitively.

**Comment by Ephemeralis** [comment]
I've attempted to squash the newspost down into "this is clicker's choice, try it out for the first time by voting in the top 20 thing we always do". I think this makes a bit more sense as a general target, and I've also made the links a bit more obvious.

File: news/2021-12-27-clickers-choice-2021.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+date: 2021-12-27 09:00:00 +0000
+---
+
+Welcome aboard a brand new voting platform by helping decide who the top 20 best osu! players of 2021 are!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo play and tournaments. As the year draw to a close, we're sending things off with the **9th iteration of the international top 20 voting**, hosted courtesy of my latest project: [Clicker's Choice](https://choice.hwc.hr/)!
+
+For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything.  This new platform should make it easier across the board to participate in such things, for users and hosts alike.
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:13]:
```suggestion
For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything. This new platform should make it easier across the board to participate in such things, for users and hosts alike.
```

File: news/2021-12-27-clickers-choice-2021.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+This year's Top 20 vote will be run on the Clicker's Choice website, which is already home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:15]: there's another instance of this on line 11

```suggestion
This year's top 20 vote will be run on the Clicker's Choice website, which is already home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```

File: news/2021-12-27-clickers-choice-2021.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Welcome aboard a brand new voting platform by helping decide who the top 20 best osu! players of 2021 are!
```
Comment (hallowatcher) [inline news/2021-12-27-clickers-choice-2021.md:7]:
```suggestion
Welcome aboard a brand new platform to help you vote for the top 20 osu! players of 2021!
```

I would suggest changing the catchphrase, it's kind of weirdly structured @Ephemeralis

File: news/2021-12-27-clickers-choice-2021.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Welcome aboard a brand new platform to help you vote for the top 20 osu! players of 2021!
```
Comment (peppy) [inline news/2021-12-27-clickers-choice-2021.md:7]: Probably change this line to read less liek this is an official project..

Comment (Ephemeralis) [reply news/2021-12-27-clickers-choice-2021.md:7]: Drawing a total blank on wording for this one. Very open to suggestions.

Comment (Walavouchey) [reply news/2021-12-27-clickers-choice-2021.md:7]: Had a long think on this one - brainstorming sure can be tedious sometimes. Maybe some combination of these will work.

```suggestion
Take part in helping the community determine the top 20 osu! players of 2021!
```

```suggestion
Come and chip in with your take on who were the best osu! players of 2021!
```

```suggestion
It's time for everyone to decide on who were the best osu! players of 2021!
```

```suggestion
Take part in helping everyone collectively identify the best osu! players of 2021!
```

```suggestion
Come on in and vote for your favorite circle clickers of the past year!
```

**Comment by peppy** [comment]
went with the one that felt the best.

**Comment by peppy** [comment]
I've applied some further rewording. Please check it. I'm okay otherwise.

**Review by peppy — APPROVED** [review]

File: news/2021-12-27-clickers-choice-2021.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Take part in helping the community determine the top 20 osu! players of 2021!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo play and tournaments. As the year draws to a close, we're sending things off with the **9th iteration of the international top 20 voting**, at the community run [Clicker's Choice](https://choice.hwc.hr/) system!
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:11]:
```suggestion
2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo play and tournaments. As the year draws to a close, we're sending things off with the **9th iteration of the international top 20 voting**, using the community-run [Clicker's Choice](https://choice.hwc.hr/) website!
```

File: news/2021-12-27-clickers-choice-2021.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything. This year's top 20 vote will be run on the Clicker's Choice (community run) website, which is also home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```
Comment (Walavouchey) [inline news/2021-12-27-clickers-choice-2021.md:13]: I get that no official status should be implied, but there's really no need to state it twice.

```suggestion
For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything. This year's top 20 vote will be run on the Clicker's Choice website, which is also home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```

Comment (hallowatcher) [reply news/2021-12-27-clickers-choice-2021.md:13]: gotta wait for @peppy or @Ephemeralis on this one as they decide when to write community-run or not, but once this is solved everything should be ready

**Comment by hallowatcher** [comment]
Text looks good, ready for merge from our side 👍

**Review by peppy — DISMISSED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**